src/Sarif.WorkItemFiling/Sarif.WorkItemFiling.csproj

```diff
@@ -22,4 +29,10 @@
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="16.143.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Octokit">
+      <HintPath>..\..\refs\Octokit.dll</HintPath>
```

Comment:
>Octokit [](start = 27, length = 7)

Can we reference this from nuget.org as a package or is it because this particular one is signed and can only work for us?

Reply:
Right, we cannot reference from NuGet as this version is unsigned. I recompiled and signed this version using our checked in key.

---
In reply to: [376503279](https://github.com/microsoft/sarif-sdk/pull/1746#discussion_r376503279) [](ancestors = 376503279)